.github/workflows/native-unix.yml

```diff
@@ -302,12 +302,12 @@ jobs:
           persist-credentials: false
       - uses: actions/setup-go@v3
         with:
-          go-version-file: 'go/adbc/go.mod'
+          go-version: 1.20.7
```

Comment:
any particular reason to specify the version explicitly instead of pull it from the `go.mod`?

Reply:
I've been through this already in prior PRs, I'm not really interested in redoing all this. (+ it's annoying trying to use an old Go since it's tied to the version of golangci-lint.)

Reply:
Reverted, though I'll have to figure out staticcheck and golangci-lint again...

Reply:
we can just bump the go.mod to be `1.20` and use the newer golangci-lint and staticcheck

Reply:
We can't, because debian etc. only have go 1.18

Reply:
again, I had to go through this all in the previous PR :grimacing: